apps/examples/board_specific_demo/ameba_wifi_csi_demo/example_wifi_csi.c

```diff
@@ -121,7 +121,7 @@ int wificsi_main(int argc, char *argv[])
 
 
 	/* unregister wifi event callback function */
-	wifi_unreg_event_handler(15, example_wifi_csi_report_cb);
+	wifi_unreg_event_handler(14, example_wifi_csi_report_cb);
```

Comment:
Let's split the commit for bsp update and other changes.

Reply:
Hi Mr.Chang, may I know which specific path do you refer to for the bsp update?
As there is a restructure on the sdk in 10.1a->10.1c update, so we integrated all of the files into one commit to prevent some compile issue.
Besides, the binaries is also included together so that the ipc api between KR4 and KM4 can match.